### DISCUSSION TITLE
Creating my Simulation Model

### DISCUSSION THREAD

**Discussion by Shaaban52** [riebl/artery#276] [Q&A]
I installed the virtual machine containing Artery and tested it by running any existing examples, and it works. Then I tried to create my own simulation model, but it seems it is not easy for me to do that. I tried to find documentation explaining that but could not find anything related to how to create a simulation example. 

Could you please help me with that?

**Comment by riebl** [comment]
The easiest way is to use one of the existing scenarios and derive your own from it. Our *highway-police* scenario is a pretty good showcase for a custom scenario. You can simply copy this directory to *scenarios/your-sim* or whatever name you like and rename the "highway-police" occurrences by a unique name for your model. Don't forget to add your directory to *scenarios/CMakeLists.txt* so it gets built and targets like "run_highway_police* (or "run_your_sim" respectively) are enabled.

You are right, we should add a section to our website about adding own simulation scenarios.

**Reply by Shaaban52** [reply]
Perfect, that's help a lot
